docs/source/yelpsoa_configs.rst

```diff
@@ -127,6 +135,83 @@ Marathon, Tron, or ``paasta remote-run``.
     a host is both whitelisted and blacklisted, the blacklist will take precedence.  Only one location type
     of whitelisting may be specified.
 
+  * ``pool``: The pool of machines a PaaSTA app runs in. If no pool is set,
+    an app will automatically be set to run in ``default`` pool.
+
+    Warning: In order for an service to be launched in a particular pool, there
+    *must* exist some nodes that already exist with that particular
+    pool attribute set.
+
+.. _k8s-placement-options:
+
+Kubernetes
+^^^^^^^^^^
+
+These options are only applicable to tasks scheduled on Kubernetes.
+
+  * ``node_selectors``: A map of labels a node is required to have for a task
+    to be launched on said node. There are several ways to define a selector.
+    The simplest is a key-value pair. For example, this selector restricts a
+    task to c3.8xlarge instances::
+
+      node_selectors:
+        instance_type: c3.8xlarge
+
+    The value can also be a list of multiple values. For example, this selector
+    restricts a task to both c3.8xlarge and m5.2xlarge instances::
+
+      node_selectors:
+        instance_type: ["c3.8xlarge", "m5.2xlarge"]
+
+    For more complex cases, an operator, and optionally a value (or values) must
+    be set. Here are some examples:
+
+    * Restricts a task to instances that are not c3.8xlarges::
+
+        node_selectors:
+          instance_type:
+            - operator: NotIn
+              values: ["c3.8xlarge"]
+
+    * Requires that a node have the ``vip`` label::
+
+        node_selectors:
+          vip:
+            - operator: Exists
+
+    * Requires that a node not have the ``vip`` label::
+
+        node_selectors:
+          vip:
```

Comment:
These are great examples, thank you for implementing this feature and for the detailled documentation :-)
As you know at Yelp we actually have a pool called vip, so may I suggest that for this specific example we use something different than 'vip', just to avoid any possible confusion (in case someone reads the docs)
A typical real world use case, and one of my dream, is to be able to specify that you want to run on a node that has fast nvme SSD storage. While we could use a list of instance types as constraint (what we currently do), I think this is a great example to use a label 'ssd' or 'fast_storage' . Can the doc use this as an example?